layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,93 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/dipesh-rawat/downloadkubernetes/integrate-k8s-website/dist/release_binaries.json" }}
+
+{{ $Binaries := slice }}
+{{ $AllOSes := slice }}
+{{ $AllArch := slice }}
+{{ $AllVersions := slice }}
+
+{{ range $key, $value := $response }}
+    {{ if eq $key "Binaries" }}
+        {{ $Binaries = $value }}
+    {{ else if eq $key "AllOSes" }}
+        {{ $AllOSes = $value }}
+    {{ else if eq $key "AllArch" }}
+        {{ $AllArch = $value }}
+    {{ else if eq $key "AllVersions" }}
+        {{ $AllVersions = $value }}
+    {{ end }}
+{{ end }}
+
+<link href="{{ "css/release_binaries.css" | relURL }}" rel="stylesheet">
+<script defer src="{{ "js/release_binaries.js" | relURL }}"></script>
+
+<div class="centered">
+    <div class="left-level">
+        <div>
+            <b>Operating Systems</b>
+            <div class="buttons" id="os">
+                {{- range $AllOSes }}
+                <button class="button" data-os="{{.}}">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
+    <div class="right-level">
+        <div>
+            <b>Stable Versions</b>
+            <div class="buttons" id="version">
+                {{- range $AllVersions }}
+                <button class="button" data-version="{{.}}">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
+
+    <div class="center-level">
+        <div>
+            <b>Architectures</b>
+            <div class="buttons" id="arch">
+                {{- range $AllArch }}
+                <button class="button" data-arch="{{.}}">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
+</div>
+<div class="table-container">
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Version</th>
+                <th>Operating System</th>
+                <th>Architecture</th>
+                <th>Download Binary</th>
+                <th>Copy Link</th>
+            </tr>
+        </thead>
+        <tbody>
+            {{- range $index, $binary := $Binaries -}}
+            {{ $Link := printf "dl.k8s.io/%s/bin/%s/%s/%s" $binary.Version $binary.OperatingSystem $binary.Architecture $binary.Name }}
+            <tr class="{{ $binary.Version }} {{ $binary.OperatingSystem }} {{ $binary.Architecture }} {{ $binary.Name }}">
+                <td>{{ $binary.Version }}</td>
+                <td>{{ $binary.OperatingSystem }}</td>
+                <td>{{ $binary.Architecture }}</td>
```

Comment:
Is there a way for us to make this localizable? Even if we don't do it initially, we should know the scope of what we can do in the future.